buildstockbatch/postprocessing.py

```diff
@@ -437,17 +561,13 @@ def combine_results(fs, results_dir, cfg, do_timeseries=True):
 
             fs.makedirs(ts_out_loc)
             logger.info(f'Created directory {ts_out_loc} for writing.')
-
-            # Read the timeseries into a dask dataframe
-            read_and_concat_ts_pq_d = dask.delayed(
-                # fs, all_cols, output_dir, filenames, group_id
-                partial(read_and_concat_enduse_timeseries_parquet, fs, all_ts_cols_sorted, ts_out_loc)
-            )
-            group_ids = list(range(npartitions))
             with tempfile.TemporaryDirectory() as tmpdir:
                 tmpfilepath = Path(tmpdir, 'dask-report.html')
                 with performance_report(filename=str(tmpfilepath)):
-                    dask.compute(map(read_and_concat_ts_pq_d, ts_files_in_each_partition, group_ids))
+                    full_df.to_parquet(ts_out_loc,
+                                       partition_on=partition_columns,
+                                       write_index=True,
+                                       name_function=lambda i: f"group{i}.parquet")
```

Comment:
Cool. I didn't know you could give a name function like that. Also, nice job on writing from a dask dataframe.

Reply:
Well, it was nice when it lasted. This no longer works when dealing with massive dataframe!